### PR TITLE
Upgrade CC components

### PR DESCRIPTION
**Pull request by jakubch1**
## Description
Please add a meaningful description for this change.
Ensure the PR has required unit tests.

## Related issue
Kindly link any related issues. E.g. Fixes #xyz.

### CODE REVIEW

File: src/package/sign/sign.proj [hunk 1]
```diff
@@ -131,11 +131,9 @@
 
       <!-- DataCollector related dlls -->
       <AssembliesToSign Include="$(ArtifactsDirectory)Microsoft.IntelliTrace.Core.dll" />
```
Comment (fhnaseer) [inline src/package/sign/sign.proj:133]: Not related to this, but I think we get these from VS repo, and they should be already signed as well,

**Review by fhnaseer — APPROVED** [review]

File: src/package/sign/sign.proj (hunk 1 above)
Comment (jakubch1) [reply src/package/sign/sign.proj:133]: @Sanan07  please try to cleanup this file after you get new packages from VS

**Review by fhnaseer — APPROVED** [review]

File: src/package/sign/sign.proj (hunk 1 above)
Comment (Sanan07) [reply src/package/sign/sign.proj:133]: What do you mean by cleanup? Do you mean clean packages, like git clean -Xdf?

Comment (jakubch1) [reply src/package/sign/sign.proj:133]: No, I mean remove files from this file (sign.proj) which are already signed by VS. Signing takes a lot of time and we don't need that

**Pull request merged**